src/migrations/20230110204839-remove-goals-objectives-feature-flag.js

```diff
@@ -0,0 +1,46 @@
+const FLAGS = ['grantee_record_page'];
```

Comment:
I don't think we need this flag anymore either

Reply:
57ae058fc322c614bb24c47309cf5f4974c72f58